### PR TITLE
NO-ISSUE: Use the SonataFlow icon in the nav bar

### PR DESCRIPTION
**Pull request by wmedvede**
**Description:**

Please make sure that your PR meets the following requirements:

- [X] You have read the [contributions doc](https://github.com/apache/incubator-kie-kogito-docs/blob/main/CONTRIBUTING.md)
- [X] Pull Request title is properly formatted: `Issue-XYZ Subject`
- [X] Pull Request title contains the target branch if not targeting main: `[0.9.x] Issue-XYZ Subject`
- [X] The nav.adoc file has a link to this guide in the proper category
- [X] The index.adoc file has a card to this guide in the proper category, with a meaningful description

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

File: antora/supplemental-ui/css/navbar.css [hunk 1]
```diff
@@ -1,3 +1,3 @@
 .navbar {
-    background: #085870 !important;
+    background: #0476d0 !important;
```
Comment (ricardozanini) [inline antora/supplemental-ui/css/navbar.css:2]: @wmedvede can you try one of these colors and see which one matches better?

> Deep Navy Blue (#002F5F) – A dark blue that matches the logo's darker tones, providing a professional and elegant look.

> Teal Blue (#007E99) – A slightly lighter shade of blue to complement the lighter parts of the logo and create a fresh, modern vibe.

> Charcoal Black (#1C1C1C) – A neutral and bold black that can make the logo stand out while maintaining simplicity.

> Soft Orange (#F4A261) – A muted orange that aligns with the logo’s orange wave, creating a warm and inviting feel.

> Cool Gray (#E0E0E0) – A neutral light gray that contrasts the logo without overshadowing it, ideal for a clean design.

AI suggested it based on our colors.

**Comment by wmedvede** [comment]
Here goes guys a screenshot of each proposal for comparison:

Proposal_1_Screenshot from 2024-12-13 17-27-41


Proposal_2_Screenshot from 2024-12-13 17-10-47


Proposal_3_Screenshot from 2024-12-13 17-15-48


Proposal_4_Screenshot from 2024-12-13 17-19-04



Proposal_5_Screenshot from 2024-12-13 17-21-38


Proposal_6_Screenshot from 2024-12-13 17-24-14

**Comment by wmedvede** [comment]
> Pretty cool, @wmedvede!! I liked the option four the most. wdyt?

great! thanks for voting.
If you don't mind, I'll add your vote for the proposal 4 in the poll!.

**Review by ricardozanini — APPROVED** [review]

**Comment by wmedvede** [comment]
I have incorporated the white circled icon kindly provided by @masha.
With or without any background color change, I think we must add it.

Examples here:

image (13)


image (14)

**Review by ricardozanini — APPROVED** [review]

**Comment by wmedvede** [comment]
The final proposals where:
Proposal1_better_icon_Screenshot from 2024-12-17 11-33-15
and
Proposal3_better_icon_Screenshot from 2024-12-17 11-37-26

Finally,  Proposal1 winned

**Pull request merged**